src/scripts/main.js

```diff
@@ -1,7 +1,22 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const message = `
+    <div class="notification ${type}">
+      <h2 class="title">${title}</h2>
+      <p class="description">${description}</p>
+    </div>`;
+
+  document.body.insertAdjacentHTML('beforeend', message);
+
+  const element = document.querySelector(`.notification.${type}`);
```

Comment:
The selector `.notification.${type}` will always select the first notification of the given type. If multiple notifications of the same type are displayed, this could lead to unexpected behavior. Consider using a more specific selector or modifying the element selection logic to target the correct notification.